shared/src/test/scala/org/adridadou/openlaw/vm/OpenlawExecutionEngineSpec.scala

```diff
@@ -35,6 +35,28 @@ class OpenlawExecutionEngineSpec extends FlatSpec with Matchers {
     }
   }
 
+  it should "run a simple clause in template and detect variable" in {
+    val text =
+      """
+        |<%
+        |[[clauseVar:LargeText]]
+        |[[clause:Clause(
```

Comment:
I don't know if it is intentional, but you need to use the variable outside of the code block to actually test it. 

Otherwise the clause will never be executed.  (actually tested)

It's a good way to check the definition piece but you will need to put it outside of the code block to test the execution piece